lib/calc-text-width.js

```diff
@@ -1,10 +1,10 @@
+const round = require('./round')
 const widthsVerdana11 = require('./widths-verdana-11.json')
 const astralRegex = require('unicode-astral-regex')
 
 const calcWidth = (charWidthTable) => {
-  const SCALE = 10 // Prevent results like 60.599999999999994
-  const widthTable = charWidthTable.map(w => Math.round(w * SCALE))
-  widthTable[64] = widthTable[64] + 6 // Slightly increase width of "@" by 0.6px
+  const widthTable = charWidthTable.map(w => round(w))
```

Comment:
No need to do this rounding on [charWidthTable](https://github.com/amio/badgen/commit/dc7f562a1c78b06a9fccf0f62c50efa1edca716f#diff-68ca46ea7eb5c942b434e942637a64d4R1).